cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion_test.go

```diff
@@ -112,7 +113,7 @@ func TestConvertToUseHyperKubeImage(t *testing.T) {
 			desc: "mismatching UnifiedControlPlaneImage tag causes an error",
 			in: &ClusterConfiguration{
 				ImageRepository:          "k8s.gcr.io",
-				KubernetesVersion:        "v1.12.0",
+				KubernetesVersion:        constants.MinimumKubernetesVersion,
```

Comment:
We don't need this one. The tested function does not depend on the current Kubernetes version.

Reply:
Thank you for the review. will revert this and all the following cases you've pointed out.